src/langsmith/agent-server-changelog.mdx

```diff
@@ -5,6 +5,10 @@ sidebarTitle: Agent Server changelog
 
 [Agent Server](/langsmith/agent-server) is an API platform for creating and managing agent-based applications. It provides built-in persistence, a task queue, and supports deploying, configuring, and running assistants (agentic workflows) at scale. This changelog documents all notable updates, features, and fixes to Agent Server releases.
 
+<a id="2025-11-13"></a>
+## v0.5.13
+- Supported singular forms for configurable headers to align with the documented usage and prevent configuration issues.
```

Comment:
```suggestion
- Support `include` and `exclude` (plural form key for `includes` and `excludes`) since a doc incorrectly claimed support for that. Now the server accepts either.
```